### PR TITLE
fix: mweb tile getting blank when joined from 2 tabs

### PR DESCRIPTION
**Pull request by amar-1995**
### Details(context, link the issue, how was the bug fixed, what does the new feature do)

-
-

### CODE REVIEW

**Review by KaustubhKumar05 — APPROVED** [review]

File: packages/hms-video-store/src/media/tracks/HMSLocalVideoTrack.ts [hunk 1]
```diff
@@ -520,19 +520,29 @@ export class HMSLocalVideoTrack extends HMSVideoTrack {
...
-      this.nativeTrack.enabled = false;
-      HMSLogger.d(this.TAG, 'visibility hidden muting track');
-      this.replaceSenderTrack(this.nativeTrack);
+      if (this.enabled) {
+        const track = await this.replaceTrackWithBlank();
+        await this.replaceSender(track, this.enabled);
+        this.nativeTrack?.stop();
+        this.nativeTrack = track;
+      } else {
+        this.nativeTrack.enabled = false;
```
Comment (eswarclynn) [inline packages/hms-video-store/src/media/tracks/HMSLocalVideoTrack.ts:529]: Why is this required? We're in this loop because this.enabled which is essentially this.nativeTrack.enabled is already false?

**Review by KaustubhKumar05 — APPROVED** [review]

**Review by eswarclynn — APPROVED** [review]
